kommentera funktionen

kinloklee opened the pull request:
Element ligger inte jättesnyggt ännu och kommenterarena kan ligga lite tokigt. Men allt funkar annars!

kinloklee committed:
kommentera funktionen

Element ligger inte jättesnyggt ännu och kommenterarena kan ligga lite tokigt. Men allt funkar annars!

pontusnordstrom89 commented on mysite/theStash/views.py:
```diff
@@ -135,3 +155,35 @@ def signup(request):
     return render(request, 'theStash/signup.html', {'form': form})
+
+
+def post_comment_detail(request, year, month, day, post):
```
radera funktionen post_comment_detail den används inte

kinloklee committed:
Tagit bort funktion post_comment_detail

Att gilla kommenterar är tänkt att finnas, men knappen till detta är inte implementerad ännu.

pontusnordstrom89 committed:
Merge branch 'main' into Comments_function
